.github/workflows/release.yaml

```diff
@@ -18,7 +18,10 @@ permissions:
 jobs:
   build-macOS:
     runs-on: macos-latest
-
+    
+    env: 
+      # macOS 11.0 Big Sur is the newest version to support universal binaries
```

Comment:
Isn't it the first version to support universal binaries?

Reply:
eeeeh yep 😅

Reply:
Fixed; [`310fd7c` (#45)](https://github.com/NiklasEi/bevy_game_template/pull/45/commits/310fd7ca06e77f600766aca9c1f5d31babbf97de)